Automattic/wp-calypso
# Pricing page rework: Update the order of products in All Products section

#### Proposed Changes

* Updates the order of products as mentioned in p1HpG7-i4a-p2#comment-57125 by moving featured products to the end of the list.

#### Testing Instructions

It's assumed that you are proxied.

- Do any one of these
    - Click on Jetpack Cloud live link below and wait for the redirect to complete, then go to `/pricing`
    - or boot up this PR 
        - Run `git fetch && git checkout fix/jp-pricing-rework/rearrange-all-products`
        - Run `yarn start-jetpack-cloud`
        - Goto [http://jetpack.cloud.localhost:3000/pricing](http://jetpack.cloud.localhost:3000/pricing)
- Confirm that you see the products in "All Products" section same as given in the screenshot below

#### Pre-merge Checklist






Completes: 0-as-1203002889297438/f

<img width="1331" alt="Screenshot 2022-09-21 at 6 24 28 PM" src="https://user-images.githubusercontent.com/18226415/191509327-e661f50e-3fe8-41d8-b7e2-28d58941cd0d.png">


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203002889297438
